newscript.go

```diff
@@ -125,15 +147,15 @@ func blankBootstick() error {
 			break
 		}
 	}
-	fmt.Printf("Running dd to put the new image onto the desired location. \n")
+	fmt.Printf("Running cp to put the new image onto the desired location. \n")
 	if err := cp(imageLoc, location); err != nil {
 		return err
 	}
 	return nil
 }
 
 func cleanup() error {
-	filesToRemove := [...]string{linuxVersion, "linux-stable", "NiChrome", "vboot_reference"}
+	filesToRemove := [...]string{*linuxVersion, "linux-stable", "NiChrome", "vboot_reference"}
```

Comment:
you don't need to remove *linuxVersion since your git command puts it in linux-stable.